### PR TITLE
Update to v2.6.0

### PR DESCRIPTION
**Pull request by sumit0190**
This is a Snowflake package (`abs.yaml` will be added after review).

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,41 +8,50 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:9]:
```suggestion
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -8,41 +8,50 @@ package:
...
+    - cython 0.29.33
+    - setuptools
     - python
+    - wheel
     - pip
   run:
-    - python
     - {{ pin_compatible('tesseract', max_pin='x.x') }}
-    - leptonica  # [not win]
+    - leptonica >=1.7.1
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
    - leptonica >=1.71
```

Comment (skupr-anaconda) [reply recipe/meta.yaml]: https://github.com/sirfz/tesserocr/tree/v2.6.0#requirements

File: recipe/meta.yaml [hunk 3]
```diff
@@ -8,41 +8,50 @@ package:
...
     - pillow
+    - python
 
 test:
```
Comment (danpetry) [inline recipe/meta.yaml:36]: There are some upstream tests we could include

**Review by danpetry — APPROVED** [review]

**Review by skupr-anaconda — APPROVED** [review]

**Pull request merged**